documentation/source/conf.py

```diff
@@ -267,5 +266,24 @@
 # If true, do not generate a @detailmenu in the "Top" node's menu.
 #texinfo_no_detailmenu = False
 
-# For now show the todoy 
+# For now show the todos
 todo_include_todos = True
+
+# -- Extra setup for C documentation with Doxygen and breathe 
+# see also https://breathe.readthedocs.io/en/latest/readthedocs.html
+
+read_the_docs_build = os.environ.get('READTHEDOCS', None) == 'True'
+if read_the_docs_build:
+    subprocess.call('cd ..; doxygen', shell=True)
```

Comment:
`subproces.call` has a `cwd` argument which is more suited than `cd ..`. And you don't need `shell` in this case.